core/primitives/src/shard_layout.rs

```diff
@@ -152,6 +152,19 @@ impl ShardLayout {
         )
     }
 
+    /// Returns the simple nightshade layout that we use in production
+    pub fn default_simple_nightshade_layout() -> ShardLayout {
```

Comment:
we should't call it 'default' (as it suggests that this is an 'empty' layout).

how about we change this function to something like:

pub fn get_production_nightshade_layout(protocol_version) {j
if protocol_version < 48 {
  // return 1-shard layout
}  else {
  // return 4-shard layout with aurora etc.
}

}